pkg/app/server/grpcapi/api.go

```diff
@@ -310,6 +310,29 @@ func (a *API) ListApplications(ctx context.Context, req *apiservice.ListApplicat
 	}, nil
 }
 
+func (a *API) RenameApplicationConfigFile(ctx context.Context, req *apiservice.RenameApplicationConfigFileRequest) (*apiservice.RenameApplicationConfigFileResponse, error) {
+	key, err := requireAPIKey(ctx, model.APIKey_READ_WRITE, a.logger)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, appID := range req.ApplicationIds {
+		err := a.applicationStore.UpdateApplication(ctx, appID, func(app *model.Application) error {
+			if app.ProjectId != key.ProjectId {
+				return fmt.Errorf("Requested application %s does not belong to your project", appID)
```

Comment:
Honestly, in this case, we would like to return BadRequest or something like that. But it's on you to overdo it.

Reply:
You are right!
My lazy part decided to ignore that thing. But let me fix it to be right.

Reply:
@nakabonne Fixed it!